server/sonar-server/src/main/java/org/sonar/server/computation/taskprocessor/CeWorkerRunnableImpl.java

```diff
@@ -72,7 +73,7 @@ private void executeTask(CeTask task) {
     CeActivityDto.Status status = CeActivityDto.Status.FAILED;
     try {
       // TODO delegate the message to the related task processor, according to task type
-      reportTaskProcessor.process(task);
+      taskProcessorRepository.getForCeTask(task).process(task);
       status = CeActivityDto.Status.SUCCESS;
       queue.remove(task, status);
     } catch (Throwable e) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1181)